### DISCUSSION TITLE
PA anchors frame & new pattern

### DISCUSSION THREAD

**Discussion by la838** [OrcaSlicer/OrcaSlicer#98] [Ideas]
Hey mate, loving the new update! Is it possible to add anchor frames to the PA pattern just so when you clean it up there's less mess? The two vertical lines in this photo to be specific.
![image](https://user-images.githubusercontent.com/87482529/210227422-295696ea-3c89-496c-9764-b167b5f6b530.png)

Also any general thoughts on new PA pattern and if is worth implantmenting? 
https://ellis3dp.com/Print-Tuning-Guide/articles/pressure_linear_advance/pattern_method.html

**Comment by SoftFever** [comment]
I thought about using anchor frames but eventually abandoned the idea because I realized that those numbers are true headaches. The lines are easy to clean anyway.

Ellis's pattern is good. 
But at the end of the day, all these tools can give us a good and accurate result, I don't plan to include more patterns to do the same task at the moment.

One thing I should have stated in my Wiki is that I recommend using the PA tower over this line method.
I designed the PA tower so that we can see how different PA values impact corners(right angle vs acute angle) as well as `z seams` on the back. The result represents the actual printing result cause actual printing settings are applied. (except for the jerk setting)
I will update that in the future.

**Reply by la838** [reply]
Ah yeah that's fair, the numbers are a pain, I usually stop the print before the numbers are laid down and then just looked at the sliced preview to see which line is which value. Then yeah is just rip from the corner then they all come off mostly at once.

Oh that's interesting, Ellis always recommended the line patterns over the tower, but yeah I always do both to double check, cause like you say, more actual printing results with the tower.
